### PR TITLE
[MTM-58404] Documentation for dashboard manager.

### PR DESCRIPTION
**Pull request by jdreSAG**

### CODE REVIEW

**Review by DawidSAG — APPROVED** [review]

**Comment by jdreSAG** [comment]
@rahultsag can you have a look?

File: content/cockpit/dashboard-manager.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Dashboard manager shows a list of device type dashboards by aggregating all dashboards that have a device type assigned.
+It also provides information about the number of devices that have access to a particular dashboard, date of creation and last modification date.
+
+<img src="/images/users-guide/cockpit/dashboard-manager.png" name="Dashboard manager"/>
+
+
+### To add a type dashboard {#to-add-type-dashboard}
+
+Adding type dashboard is possible only from device view. To add a dashboard, refer to [To create a dashboard](/cockpit/working-with-dashboards/#to-create-a-dashboard),
```
Comment (MWindel) [inline content/cockpit/dashboard-manager.md:27]:
```suggestion
To add a new dashboard for a device type select the device from the device list in the **Groups** menu, refer to [To create a dashboard](/cockpit/working-with-dashboards/#to-create-a-dashboard) and [Dashboard template](/cockpit/working-with-dashboards/#dashboard-template) for more information. 
```

File: content/cockpit/dashboard-manager.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Dashboard manager shows a list of device type dashboards by aggregating all dashboards that have a device type assigned.
+It also provides information about the number of devices that have access to a particular dashboard, date of creation and last modification date.
+
+<img src="/images/users-guide/cockpit/dashboard-manager.png" name="Dashboard manager"/>
+
+
+### To add a type dashboard {#to-add-type-dashboard}
+
+Adding type dashboard is possible only from device view. To add a dashboard, refer to [To create a dashboard](/cockpit/working-with-dashboards/#to-create-a-dashboard), 
+and to make it type dashboard see [Dashboard template](/cockpit/working-with-dashboards/#dashboard-template).
```
Comment (MWindel) [inline content/cockpit/dashboard-manager.md:28]: delete this line

File: content/cockpit/dashboard-manager.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+### To remove type dashboard {#to-remove-type-dashboard}
+
+Removing type dashboard is possible both from device view and dashboard manager. To remove a dashboard, click **Delete** button on the corresponding row.
```
Comment (MWindel) [inline content/cockpit/dashboard-manager.md:32]:
```suggestion
To remove a device type dashboard, navigate to the device list in the **Groups** menu or the **Dashboard manager** in the **Configuration menu**. In the row of the selected dashboard, click the delete icon <i class="dlt-c8y-icon-editing-trash text-danger icon-20"></i> on the right. ```

File: content/cockpit/dashboard-manager.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+{{< c8y-admon-info >}}
+Removing type dashboard will remove it from all devices that have access to it. There is no possibility to remove a dashboard from a single device.
```
Comment (MWindel) [inline content/cockpit/dashboard-manager.md:35]: @jdreSAG Do you refer to "Removing" or "Deleting" a dashboard in this section?

to remove = just removed from the platform, can be restored easily (1-2 clicks) and fully. 
to delete = the dashboard/device/group/user/etc. is permanently and fully deleted from the platform. The data cannot be restored.
Please refer to our Style guide for more information on our Documentation terminology for UI components and actions.
https://sagportal.sharepoint.com/sites/CumulocityIoTDocs/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FCumulocityIoTDocs%2FShared%20Documents%2FGeneral%2FFormatting%20%26%20style%2FCumulocity%20Documentation%20Style%20Guide%2Epdf&viewid=a8e05faa%2D2a52%2D4b46%2Db8db%2Ddc9fb9eda4c4&parent=%2Fsites%2FCumulocityIoTDocs%2FShared%20Documents%2FGeneral%2FFormatting%20%26%20style.

Comment (MWindel) [reply content/cockpit/dashboard-manager.md:35]: For "to delete"

```suggestion
If you delete a device type dashboard, it gets deleted from all devices that have access to it. It is not possible to delete a device type dashboard from a single device.
```

Comment (MWindel) [reply content/cockpit/dashboard-manager.md:35]: For "to remove"

```suggestion
If you remove a device type dashboard, it is removed from all devices that have access to it. It is not possible to remove a device type dashboard from a single device.
```

File: content/cockpit/dashboard-manager.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+{{< /c8y-admon-info >}}
+
+### To edit type dashboard {#to-edit-type-dashboard}
+
+Click edit button on the corresponding row to open list with all devices that has access to the dashboard.
```
Comment (MWindel) [inline content/cockpit/dashboard-manager.md:40]:
```suggestion
Click the edit icon <i class="dlt-c8y-icon-edit1 text-primary icon-20"></i> on the corresponding row to open list with all devices that has access to the dashboard.
```

File: content/cockpit/dashboard-manager.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+Removing type dashboard is possible both from device view and dashboard manager. To remove a dashboard, click **Delete** button on the corresponding row.
+
+{{< c8y-admon-info >}}
+Removing type dashboard will remove it from all devices that have access to it. There is no possibility to remove a dashboard from a single device.
+{{< /c8y-admon-info >}}
+
+### To edit type dashboard {#to-edit-type-dashboard}
+
+Click edit button on the corresponding row to open list with all devices that has access to the dashboard. 
+Click **Open instance** button to navigate to dashboard view in context of the device. Then, see [To edit a dashboard](/cockpit/working-with-dashboards/#to-edit-a-dashboard) for more.
```
Comment (MWindel) [inline content/cockpit/dashboard-manager.md:41]:
```suggestion
Click the  open instance icon <i class="dlt-c8y-icon-file-create text-primary icon-20"></i> to navigate to dashboard view in context of the device. Then, see [To edit a dashboard](/cockpit/working-with-dashboards/#to-edit-a-dashboard) for more.
```

Comment (MWindel) [inline content/cockpit/dashboard-manager.md:41]:
```suggestion
In the **Select dashboard instance to edit** dialog window, select the desired device and click the open instance icon <i class="dlt-c8y-icon-file-create text-primary icon-20"></i> on the right to navigate to dashboard assigned to this device. Now you can edit the dashboard settings or widgets. See [To edit a dashboard](/cockpit/working-with-dashboards/#to-edit-a-dashboard) for more detailed information.
```

File: content/cockpit/dashboard-manager.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+<img src="/images/users-guide/cockpit/dashboard-manager-edit.png" name="Dashboard manager edit"/>
```
Comment (MWindel) [inline content/cockpit/dashboard-manager.md:43]: @jdreSAG This screenshot does not really serve a purpose. I would propose to delete it, as it only shows the dialog window.

File: content/cockpit/dashboard-manager.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+  - app_enablement
+helpcontent:
+  - label: dashboard-manager
+    title: Dashboard manager
+    content: "This dashboard manager shows a list of device type dashboards by aggregating all dashboards that have a device type assigned. 
+    
+
+Dashboards can be removed via the list. Moreover you can view all devices that have access to a particular dashboard. Dashboards can be edited by navigating to the device view."
+---
+
```
[the remark this answers is not in the document]
Comment (jdreSAG) [reply content/cockpit/dashboard-manager.md:18]: added info where you can find dashboard manager

File: content/cockpit/dashboard-manager.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+Dashboard manager shows a list of device type dashboards by aggregating all dashboards that have a device type assigned.
```
[the remark this answers is not in the document]
Comment (jdreSAG) [reply content/cockpit/dashboard-manager.md:19]: by "target asset model" we define devices (that indeed has some type and they are associated to "device type dashboards" by that type). dashboard manager in it's main view shows list of dashboards, not devices.

File: content/cockpit/dashboard-manager.md (hunk 2 above)
Comment (jdreSAG) [reply content/cockpit/dashboard-manager.md]: done

File: content/cockpit/dashboard-manager.md (hunk 7 above)
Comment (jdreSAG) [reply content/cockpit/dashboard-manager.md]: done

File: content/cockpit/dashboard-manager.md (hunk 4 above)
Comment (jdreSAG) [reply content/cockpit/dashboard-manager.md]: changed all 'remove' to 'delete'

File: content/cockpit/dashboard-manager.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+  - json
+section:
+  - app_enablement
+helpcontent:
+  - label: dashboard-manager
+    title: Dashboard manager
+    content: "This dashboard manager shows a list of device type dashboards by aggregating all dashboards that have a device type assigned. 
+    
+
+Dashboards can be deleted via the list. Moreover you can view all devices that have access to a particular dashboard. Dashboards can be edited by navigating to the device view."
```
Comment (MWindel) [inline content/cockpit/dashboard-manager.md:16]:
```suggestion
Dashboards can be deleted via the list. Moreover, you can view all devices that have access to a particular dashboard. Dashboards can be edited by navigating to the device view."
```

File: content/cockpit/dashboard-manager.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (MWindel) [reply content/cockpit/dashboard-manager.md:19]: @jdreSAG Thank you for clarifying.

File: content/cockpit/dashboard-manager.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+It also provides information about the number of devices that have access to a particular dashboard, date of creation and last modification date.
+
+To open Dashboard manager, click **Dashboard manager** in the **Configuration** menu of the navigator.
+
+<img src="/images/users-guide/cockpit/dashboard-manager.png" name="Dashboard manager"/>
+
+
+### To add a type dashboard {#to-add-type-dashboard}
+
+To add a new dashboard for a device type select the device from the device list in the **Groups** menu, refer to [To create a dashboard](/cockpit/working-with-dashboards/#to-create-a-dashboard) and [Dashboard template](/cockpit/working-with-dashboards/#dashboard-template) for more information.
```
Comment (MWindel) [inline content/cockpit/dashboard-manager.md:29]:
```suggestion
To add a new dashboard for a device type select the device from the device list in the **Groups** menu. Refer to [To create a dashboard](/cockpit/working-with-dashboards/#to-create-a-dashboard) and [Dashboard template](/cockpit/working-with-dashboards/#dashboard-template) for more information. 
```

File: content/cockpit/dashboard-manager.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+<img src="/images/users-guide/cockpit/dashboard-manager.png" name="Dashboard manager"/>
+
+
+### To add a type dashboard {#to-add-type-dashboard}
+
+To add a new dashboard for a device type select the device from the device list in the **Groups** menu, refer to [To create a dashboard](/cockpit/working-with-dashboards/#to-create-a-dashboard) and [Dashboard template](/cockpit/working-with-dashboards/#dashboard-template) for more information. 
+
+### To delete type dashboard {#to-delete-type-dashboard}
+
+To delete a device type dashboard, navigate to the device list in the **Groups** menu or the **Dashboard manager** in the **Configuration menu**. In the row of the selected dashboard, click the delete icon <i class="dlt-c8y-icon-editing-trash text-danger icon-20"></i> on the right.
```
Comment (MWindel) [inline content/cockpit/dashboard-manager.md:33]:
```suggestion
To delete a device type dashboard, navigate to the device list in the **Groups** menu or to the **Dashboard manager** in the **Configuration menu**. In the row of the selected dashboard, click the delete icon <i class="dlt-c8y-icon-editing-trash text-danger icon-20"></i> on the right.
```

**Review by MWindel — APPROVED** [review]

**Pull request merged**